rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -356,7 +356,7 @@ rsyslogd_process_names:
   - rsyslogd
 
 elasticsearch_process_names:
-  - elasticsearch
+  - java 
```

Comment:
Elasticsearch did not have a process named 'elasticsearch', so it was failing because it uses java as the process tracking.